modules/react/common/stories/Responsive.stories.mdx

```diff
@@ -0,0 +1,314 @@
+const mediaQuery = down('xl'); // Returns '@media (min-width: 0)'
+const styles = {
+  [mediaQuery]: {
+    padding: space.m,
+  },
+};
+```
+
+`between()` - Returns a media query between two given breakpoints. Given `start` and `end`
+breakpoint keys ("zero", "s", "m", "l", "xl"), this function returns a media query (string) using a
+min-width and max-width.
+
+> **Note**: This function subtracts `0.5px` from the next breakpoint value to prevent collisions.
+> For example, `breakpoints.values.s`, has a `min-width` of `320px`, and the `max-width` is
+> `767.5px`).
+
+If the `xl` breakpoint is provided, this function returns a media query with only a `min-width`, as
+seen in the second example below.
+
+```ts
+import {useTheme} from '@workday/canvas-kit-react/common';
+import {space} from '@workday/canvas-kit-react/tokens';
+const theme = useTheme();
+const {between} = theme.canvas.breakpoints;
+// Returns '@media (min-width: 320px) and (max-width: 1023.5px)'
+const mediaQuery = between('s', 'm');
+const styles = {
+  [mediaQuery]: {
+    padding: space.s,
+  },
+};
+```
+
+This example uses `xl` as the `end` breakpoint and only adds a min-width to the media query.
+
+```ts
+import {useTheme} from '@workday/canvas-kit-react/common';
+import {space} from '@workday/canvas-kit-react/tokens';
+const theme = useTheme();
+const {between} = theme.canvas.breakpoints;
+const mediaQuery = between('m', 'xl'); // Returns '@media (min-width: 768px)'
+const styles = {
+  [mediaQuery]: {
+    padding: space.s,
+  },
+};
+```
+
+`only()` - Returns a media query with a `min-width` and `max-width` for a given breakpoint. Given a
+breakpoint key ("zero", "s", "m", "l", "xl"), this function returns a media query (string) using a
+`min-width` and `max-width`.
+
+> **Note**: This function subtracts `0.5px` from the next breakpoint value to prevent collisions.
+> For example, `breakpoints.values.s`, has a `min-width` of `320px`, and the `max-width` is
+> `767.5px`).
+
+If the `xl` breakpoint is provided, this function returns a media query with only a `min-width` of
+`1440px`, as seen in the second example below.
+
+```ts
+import {useTheme} from '@workday/canvas-kit-react/common';
+import {space} from '@workday/canvas-kit-react/tokens';
+
+const theme = useTheme();
+const {only} = theme.canvas.breakpoints;
+const mediaQuery = only('s'); // Returns '@media (min-width: 320px) and (max-width: 767.5px)'
+const styles = {
+  [mediaQuery]: {
+    padding: space.s,
+  },
+};
+```
+
+This example uses the `xl` breakpoint and only adds a `min-width` of `1440px` to the media query.
+
+```ts
+import {useTheme} from '@workday/canvas-kit-react/common';
+import {space} from '@workday/canvas-kit-react/tokens';
+
+const theme = useTheme();
+const {only} = theme.canvas.breakpoints;
+const mediaQuery = only('xl'); // Returns '@media (min-width: 1440px)'
+const styles = {
+  [mediaQuery]: {
+    padding: space.s,
+  },
+};
+```
+
+### Viewport vs Container
```

Comment:
This section should be further up in the document. I'd consider putting it in the `## What is Responsive Styling?` section I suggested at the top.

Reply:
I looked at this and in terms of flow for the docs, I revamped the entire section. I added more description of container vs viewport